eslint.config.mjs

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Eslint changed their config syntax completely. It is pure JS now instead of JSON. Followed this guide: https://eslint.org/docs/latest/use/configure/migration-guide